Libraries/pbxbuild/Sources/Phase/ProductTypeResolver.cpp

```diff
@@ -335,6 +465,15 @@ resolve(Phase::Environment const &phaseEnvironment, Phase::Context *phaseContext
     }
 
     /*
+     * Create the product's module maps, if required.
+     */
+    if (pbxsetting::Type::ParseBoolean(environment.resolve("DEFINES_MODULE"))) {
```

Comment:
The documentation helps clear things up for me, thanks.  I now the difference between #include and #import's behaviors!